install/helm/space-agon/values.yaml

```diff
@@ -21,18 +21,20 @@ frontend:
   image: 
     repository: local/space-agon-frontend
     tag: TAG
-    imagePullPolicy: IfNotPresent
+    imagePullPolicy: Always
+  omCoreAddress: "https://om-core-976869741551.us-central1.run.app"
```

Comment:
Dummy address would be better(i.e. localhost).

Reply:
Done. I used `YOUR_OM_CORE_ADDRESS` because I think using `localhost` can be a bit misleading, leading users to believe they should deploy the `om-core` service locally instead of on Cloud Run.

Reply:
Thank you! Confirmed.